### DISCUSSION TITLE
Re-ASSETS

### DISCUSSION THREAD

**Discussion by gianlore** [Keriew/augustus#441] [General]
Sorry if I take up the topic "assets", but in my opinion there are problems.
With the augustus-3.0.1.80-2229b53b-windows version I downloaded assets-3.0.1.82-f7bceff3-development I copied it to the Caesar 3 directory, and everything worked. In the augustus-log.txt file I find the line:
INFO: Asset path detected at C: \ SIERRA \ Caesar3 \ / assets.
INFO: Model loaded
INFO: Loading assetlist file aesthetics.xml
INFO: Loading assetlist file aesthetics_lizzaran.xml
INFO: Loading assetlist file architect.xml
INFO: Loading assetlist file ceres_temple.xml
INFO: Loading assetlist file circus.xml
etc. etc.
I downloaded augustus-3.0.1.105-5f0c8bb0-windows and copied the files in the Caesar 3 directory, starting augustus.exe I find this window:
![Avviso 02](https://user-images.githubusercontent.com/72648521/123161422-a6cf6480-d46f-11eb-82ed-51f47594a375.jpg)

which is equivalent to this:
![Avviso 01](https://user-images.githubusercontent.com/72648521/123161463-b484ea00-d46f-11eb-84e9-a716254d6466.jpg)

In the main Caesar 3 directory the "assets" directory exists and in the augustus-log.txt file I find the following lines:

INFO: Asset path detected at C: \ SIERRA \ Caesar3 \ / assets
ERROR: Unable to open png file Color_Cursors / Arrow.png
ERROR: Unable to open png file Color_Cursors / Arrow.png
ERROR: Unable to open png file Color_Cursors / Shovel.png
ERROR: Unable to open png file Color_Cursors / Shovel.png
ERROR: Unable to open png file Color_Cursors / Sword.png
ERROR: Unable to open png file Color_Cursors / Sword.png
INFO: Model loaded
INFO: Loading assetlist file aesthetics.xml
ERROR: Error parsing file aesthetics.xml
etc. etc.

How can this problem be remedied?
Thank you

**Comment by gianlore** [comment]
Sorry for the message, but I solved it !!!

**Reply by jraphae** [reply]
I'm having the same issue, can you please explain how to solve it? 
thanks a lot

**Reply by crudelios** [reply]
@jraphae, if you are using the latest unstable build of Augustus, follow these instructions:

Fully delete the `assets` folder from the C3 install folder and/or the `augustus.exe` location.

Then grab the new `assets` folder from https://augustus.josecadete.net/download/latest/development/assets and copy it to the same folder where you have `augustus.exe` (NOT the C3 dir anymore)

**Reply by jraphae** [reply]
thanks for you response. 

I've done exactly as you said, and unfortunately it still isn't working. 
I'm not very tech savvy, but here is all i've done:
1- downloaded augustus
2- unzipped, and pasted it in the C3 folder (GOG game)
3- deleted the assets folder, and replaced with the one you've linked

I still get the same error message (extra assets couldn't be loaded)

**Reply by crudelios** [reply]
Which version of Augustus do you have? Check on the main menu.

**Reply by VentureOn** [reply]
Hi Crudelios, new user here as well having the same issue. I also followed the above walkthrough and still no luck.

> Which version of Augustus do you have? Check on the main menu.

Augustus v3.0.1

**Reply by VentureOn** [reply]
I will add, I've also noticed the "theatre's" I've placed now appear as a black square. I can't right click them, delete, them, just dead space now. Placing a new theatre (before saving and exiting out) does not create the black square, but it does not "appear" and distorts the grass where I placed it. See attached, the space next to the market is where I just placed a new theatre. 
![Capture](https://user-images.githubusercontent.com/86526188/123517388-9d601b00-d66e-11eb-911d-c6593155cb66.PNG)

**Reply by jraphae** [reply]
@crudelios same for me v.3.0.1.106

@VentureOn I have the same issue with roadblocks, theatres, etc...

**Comment by crudelios** [comment]
Ok, @VentureOn, for v3.0.1 you need the default assets that come with that version. So delete the `assets` folder and redownloadthe one from v3.0.1

@jraphae On the other hand, you do need the development assets, so follow my previous instructions.

**Reply by jraphae** [reply]
@crudelios It worked, thanks alot for your help!
